web/src/input_pill.ts

```diff
@@ -19,6 +19,8 @@ export type InputPillItem<T> = {
     deactivated?: boolean;
     status_emoji_info?: EmojiRenderingDetails & {emoji_alt_code?: boolean}; // TODO: Move this in user_status.js
     should_add_guest_user_indicator?: boolean;
+    user_id?: number;
+    id?: number;
```

Comment:
Original Review:
> sahil: Why not group_id here instead of id?

Hi Sahil, it's because `user_group_pill.ts` used `id` instead if `group_id`. I also agree that it be `group_id`, consistent with `user_id` instead of just `id`. I can add it as a prep commit if you think we should make the change

_relevant code in_ `user_group_pill.ts`: 
![image](https://github.com/zulip/zulip/assets/108744694/ab64ddee-eb40-4838-9e67-4f21955a1e58)